docs/src/modules/components/ComponentPageTabs.js

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { useRouter } from 'next/router';
+import Box from '@mui/material/Box';
+import Tabs from '@mui/material/Tabs';
+import Tab from '@mui/material/Tab';
+import Link from 'docs/src/modules/components/Link';
+import { useTranslate } from 'docs/src/modules/utils/i18n';
+
+export default function ComponentPageTabs(props) {
+  const {
+    markdown: { headers },
+    activeTab,
+    setActiveTab,
+    children,
+  } = props;
+
+  return (
+    <Box
+      className="component-tabs"
+      sx={{
+        position: 'sticky',
+        top: 5,
+        zIndex: 9999,
+      }}
+    >
+      <Tabs
+        value={activeTab}
+        onChange={(e, value) => setActiveTab(value)}
+        sx={{ fontSize: 'large' }}
```

Comment:
To avoid sticky ad 🙂

```suggestion
      sx={{ display: 'inline' }}
    >
      <Tabs
        value={activeTab}
        onChange={(e, value) => setActiveTab(value)}
        sx={{
          fontSize: 'large',
          position: 'sticky',
          top: 5,
          zIndex: 9999,
        }}
```

Reply:
Thanks!